core/embed/rust/src/ui/model_mercury/flow/util.rs

```diff
@@ -262,9 +269,19 @@ impl ConfirmBlobParams {
         }
         .into_paragraphs();
 
+        let confirm_extra = if self.cancel {
```

Comment:
so `cancel` param means it's *just* "Cancel" in the menu but if `cancel == false` then the menu contains <Info verb> *and* "Cancel"?

Reply:
"Cancel" is always there - whether in the menu or as an "x" (if the menu is unnecessary because it just has one option). Perhaps this should be changed, such that it is the case everywhere ... ? Basically there should be no menu with just "cancel" in it? I guess it makes no sense. I did not want to touch other flows here, but I think your concern makes sense, that this is confusing.